demos/groktunnel/main.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+	"net/http/httputil"
+	"strings"
+	"sync"
+	"time"
+
+	vhost "github.com/inconshreveable/go-vhost"
+	shortuuid "github.com/lithammer/shortuuid/v3"
+	"github.com/progrium/qmux/golang/session"
+)
+
+func fatal(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func main() {
+	var port = flag.String("p", "9999", "server port to use")
+	var host = flag.String("h", "vcap.me", "server hostname to use")
+
+	flag.Parse()
+
+	if flag.Arg(0) != "" {
```

Comment:
Did you want to provide some usage docs in the source, or maybe a README? I guess it looks like this:
```
usage:
# start server:
go run main.go
# expose port 8080:
go run main.go 8080
```

Reply:
yea will add